pkg/gui/controllers/branches_controller.go

```diff
@@ -187,6 +187,7 @@ func (self *BranchesController) viewUpstreamOptions(selectedBranch *models.Branc
 				ShowBranchHeads:         false,
 			})
 		},
+		Key: 'v',
```

Comment:
Huh? Did this come back accidentally?

Reply:
Yes it did haha. I re-introduced it to ensure that if we wanted, we could set 'v' as a key in a menu item. I'll remove it again given that we don't want to do that.